larray/core/session.py

```diff
@@ -24,6 +25,26 @@ class Session(object):
         Name of file to load or dictionary containing couples (name, array).
     kwargs : dict of str, array
         List of arrays to add written as 'name'=array, ...
+        
+    Examples
+    --------
+    >>> arr1, arr2, arr3 = ndtest((3, 3)), ndtest((6, 4)), ndtest((2, 8))
+    
+    create a Session using keyword arguments
+    
+    >>> s = Session(arr1=arr1, arr2=arr2, arr3=arr3)
+    
+    create a Session by passing a dictionary (but you lose order)
+    
+    >>> s = Session({'arr1': arr1, 'arr2': arr2, 'arr3': arr3})
+     
+    create a Session by passing an ordered dictionary
+    
+    >>> s = Session(OrderedDict([('arr1', arr1), ('arr2', arr2), ('arr3', arr3)]))
```

Comment:
to conserve ordering I would mention this instead (in other examples too):

s = Session([('arr2', arr2), ('arr1', arr1), ('arr3', arr3)])